examples/flutter_view/android/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="app_name">Flutter View</string>
+  <string name="title">Flutter Application</string>
+  <string name="increment">+</string>
+  <string name="button_tap">Flutter button tapped 0 times.</string>
```

Comment:
The text is duplicated in the Java code. I believe the resources XML supports both placeholders (for the tap count) and singular vs plural versions, see https://developer.android.com/guide/topics/resources/string-resource.html